lib/services.key.js

```diff
@@ -203,9 +202,7 @@ function addRoutes(app) {
           identityId, req.params.publicKey);
 
         // get public key
-        brIdentity.getPublicKey({id: publicKeyId}, function(err, key) {
-          callback(err, key);
-        });
+        brIdentity.getPublicKey({id: publicKeyId}, callback);
```

Comment:
Looks like another case where the callback was required before because of extra params being sent after `key`.